build-ps/debian/rules_zenfs

```diff
@@ -0,0 +1,218 @@
+		-DINSTALL_INCLUDEDIR=include/mysql \
+		-DINSTALL_INFODIR=share/mysql/docs \
+		-DINSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH) \
+		-DINSTALL_MANDIR=share/man \
+		-DINSTALL_MYSQLSHAREDIR=share/mysql \
+		-DINSTALL_MYSQLTESTDIR=lib/mysql-test \
+		-DINSTALL_PLUGINDIR=lib/mysql/plugin/debug \
+		-DINSTALL_SBINDIR=sbin \
+		-DINSTALL_SCRIPTDIR=bin \
+		-DINSTALL_SUPPORTFILESDIR=share/mysql \
+		-DSYSCONFDIR=/etc/mysql \
+		-DMYSQL_UNIX_ADDR=/var/run/mysqld/mysqld.sock \
+		-DCMAKE_BUILD_TYPE=Debug \
+		-DCOMPILATION_COMMENT=$(COMPILATION_COMMENT_DEBUG) \
+		-DROCKSDB_PLUGINS=zenfs \
+		-DSYSTEM_TYPE="debian-linux-gnu" \
+		-DINSTALL_LAYOUT=DEB \
+		-DWITH_INNODB_MEMCACHED=ON \
+		-DWITH_MECAB=system \
+		-DWITH_ARCHIVE_STORAGE_ENGINE=ON \
+		-DWITH_BLACKHOLE_STORAGE_ENGINE=ON \
+		-DWITH_FEDERATED_STORAGE_ENGINE=ON \
+		-DWITH_PAM=ON \
+		-DWITH_ROCKSDB=ON \
+		-DROCKSDB_DISABLE_AVX2=1 \
+		-DROCKSDB_DISABLE_MARCH_NATIVE=1 \
+		-DFORCE_INSOURCE_BUILD=1 \
+		-DDOWNLOAD_BOOST=1 \
+		-DWITH_BOOST=libboost \
+		-DWITH_SYSTEM_LIBS=ON \
+                -DWITH_PROTOBUF=bundled \
+                -DWITH_RAPIDJSON=bundled \
+                -DWITH_ICU=bundled \
+                -DWITH_ZSTD=bundled \
+		-DWITH_ZLIB=bundled \
+                -DWITH_LZ4=bundled \
+                -DWITH_EDITLINE=bundled \
+                -DWITH_LIBEVENT=bundled \
+                -DWITH_NUMA=ON \
+                -DWITH_LDAP=system \
+		-DWITH_EXTRA_CHARSETS=all $(TOKUDB_OPTS_DEBUG) ..
+endif
+
+	( test -d $(builddir) || mkdir $(builddir) ) && cd $(builddir) && \
+	    cmake -DBUILD_CONFIG=mysql_release \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DINSTALL_DOCDIR=share/mysql/docs \
+		-DINSTALL_DOCREADMEDIR=share/mysql \
+		-DINSTALL_INCLUDEDIR=include/mysql \
+		-DINSTALL_INFODIR=share/mysql/docs \
+		-DINSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH) \
+		-DINSTALL_MANDIR=share/man \
+		-DINSTALL_MYSQLSHAREDIR=share/mysql \
+		-DINSTALL_MYSQLTESTDIR=lib/mysql-test \
+		-DINSTALL_PLUGINDIR=lib/mysql/plugin \
+		-DINSTALL_SBINDIR=sbin \
+		-DINSTALL_SCRIPTDIR=bin \
+		-DINSTALL_SUPPORTFILESDIR=share/mysql \
+		-DSYSCONFDIR=/etc/mysql \
+		-DMYSQL_UNIX_ADDR=/var/run/mysqld/mysqld.sock \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCOMPILATION_COMMENT=$(COMPILATION_COMMENT_RELEASE) \
+		-DROCKSDB_PLUGINS=zenfs \
+		-DSYSTEM_TYPE="debian-linux-gnu" \
+		-DINSTALL_LAYOUT=DEB \
+		-DWITH_INNODB_MEMCACHED=ON \
+		-DFORCE_INSOURCE_BUILD=1 \
+		-DWITH_MECAB=system \
+		-DWITH_ARCHIVE_STORAGE_ENGINE=ON \
+		-DWITH_BLACKHOLE_STORAGE_ENGINE=ON \
+		-DWITH_FEDERATED_STORAGE_ENGINE=ON \
+		-DWITH_PAM=ON \
+		-DWITH_ROCKSDB=ON \
+		-DROCKSDB_DISABLE_AVX2=1 \
+		-DROCKSDB_DISABLE_MARCH_NATIVE=1 \
+		-DDOWNLOAD_BOOST=1 \
+		-DWITH_BOOST=libboost \
+		-DWITH_SYSTEM_LIBS=ON \
+                -DWITH_PROTOBUF=bundled \
+                -DWITH_RAPIDJSON=bundled \
+                -DWITH_ICU=bundled \
+                -DWITH_ZSTD=bundled \
+                -DWITH_LZ4=bundled \
```

Comment:
indebtation

Reply:
Please compare with rules, you will see that there's no indentation problem, sent screenshot into slack